composer.json

```diff
@@ -37,7 +37,8 @@
         "php-http/mock-client": "~0.1",
         "symfony/process": "^2.3||^3.0",
         "symfony/http-kernel": "^2.3||^3.0",
-        "puli/composer-plugin": "^1.0.0-beta9@beta"
+        "puli/composer-plugin": "^1.0.0-beta9@beta",
+        "phpunit/phpunit": "<5.4"
```

Comment:
we have one build with composer --prefer-lowest to check that too old versions are not ok. this should probably be `^5.0.0 <5.4` to avoid too old versions of phpunit.